src/sentry/api/serializers/models/integration.py

```diff
@@ -24,7 +24,7 @@ def serialize(self, obj, attrs, user):
 
 
 class IntegrationConfigSerializer(IntegrationSerializer):
-    def serialize(self, obj, attrs, user):
+    def serialize(self, obj, attrs, user, organization_id=None):
```

Comment:
i think we usually pass context kwargs to `__init__` rather than serialize. example: https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/tagvalue.py#L19